repo-policy/main.tf

```diff
@@ -36,18 +36,18 @@ module "tyk-identity-broker" {
 { branch    = "master",
 	reviewers = "1",
 	convos    = "false",
-	required_tests = ["1.16"]},
+	required_tests = ["1.19"]},
 { branch    = "release-1.2",
 	reviewers = "0",
 	convos    = "false",
 	required_tests = ["1.15"]},
 { branch    = "release-1.3",
 	reviewers = "0",
 	convos    = "false",
-	required_tests = ["1.16"]},
+	required_tests = ["1.19"]},
```

Comment:
@mativm02 changing the required tests in release-1.3 is not needed since we are not going to propagate this changes to release-1.3

Reply:
thanks tom!